x-pack/test/security_solution_api_integration/test_suites/entity_analytics/default_license/risk_engine/risk_engine_privileges.ts

```diff
@@ -197,5 +198,41 @@ export default ({ getService }: FtrProviderContext) => {
         });
       });
     });
+
+    describe('Risk engine init API privilege check', () => {
```

Comment:
I have put these tests here because they only test privileges, and this file creates users for testing privileges, it seemed more logical (and efficient) to have them all here.

Reply:
By "efficient" do you mean something other than "in the same file"?

My only real argument against this is that it's surprising: if I'm looking for an integration test about auth requirements for the `init` API, and I find the `init` API test file, I would assume its absence there means there are no requirements, not that the test exists elsewhere. Of course, if things change the proper test is still going to go red, but I'm speaking more to the discoverability/documentation aspect of tests.

Reply:
I meant efficient in the sense that we create the necessary test users already in this file for testing risk engine privileges. Coming back to this after not looking for a couple of days it is probably less logical than I first thought, but I do think its not unacceptably lazy to keep them here :D